app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -95,6 +106,20 @@ class StateFragmentPresenter @Inject constructor(
   private fun subscribeToCurrentState() {
     ephemeralStateLiveData.observe(fragment, Observer<EphemeralState> { result ->
       logger.d("StateFragment", "getCurrentState: ${result.state.name}")
+      entity_type = "exploration"
+      // TODO replace exploration Id from result. Exploration id is missing in proto
```

Comment:
If possible, link it to a issue number otherwise write it like this:
`// TODO(Veena): Replace exploration id from result, currently proto is missing the exploration id.`

Reply:
Fetch exploration Id from Exploration activity.